app/controllers/application.js

```diff
@@ -14,9 +14,4 @@ export default Controller.extend({
   didRender() {
     this.set('institution', this.store.find('institution'));
   },
```

Comment:
Looks odd that this finishes on a comma - I realize it did before so I'm going to merge anyway, but wondering if this is correct!